stdlib/public/Concurrency/AsyncStream.swift

```diff
@@ -477,15 +478,21 @@ extension AsyncStream.Continuation.YieldResult: Sendable where Element: Sendable
 
 @available(SwiftStdlib 6.2, *)
 extension AsyncStream.Continuation: Hashable {
+  @inlinable
   @available(SwiftStdlib 6.2, *)
+  @inlinable
   public func hash(into hasher: inout Hasher) {
     return hasher.combine(ObjectIdentifier(storage))
   }
+  @inlinable
   @available(SwiftStdlib 6.2, *)
+  @inlinable
```

Comment:
nit: looks like there are double annotations for some reason – is that intentional?

Reply:
that's a cherry pick gone wrong